package.json

```diff
@@ -269,6 +283,13 @@
 					"when": "view == inspect-node && viewItem == editable_value",
 					"group": "inline"
 				}
+			],
+			"explorer/context": [
+				{
+					"command": "godot-tool.set_scene_file",
+					"when": "resourceLangId == properties && resourceExtname == .tscn",
```

Comment:
There was a similar comment here where forcing `.tscn` files to be the only thing you can run to be an issue when I first implemented launching a scene file, to be able to launch scenes that do are _not_ `.tscn` files.